build/configs/rtl8720e/hello/defconfig

```diff
@@ -262,7 +262,7 @@ CONFIG_ARCH_BOARD_HAVE_FLASH=y
 CONFIG_FLASH_MINOR=0
 CONFIG_FLASH_PART_SIZE="32,28,16,4,432,3068,3068,512,1024,8,"
 CONFIG_FLASH_PART_TYPE="none,none,none,none,none,kernel,kernel,smartfs,none,bootparam,"
-CONFIG_FLASH_PART_NAME="bl,reserved,ftl,system_data,ss,kernel,kernel,userfs,samsung_reserved,bootparam,"
```

Comment:
Spell mistake in commit message bootlaoders => bootloader

Reply:
Corrected, thanks